pkg/cmd/server/api/v1/conversions.go

```diff
@@ -127,6 +127,9 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			}
 		},
 		func(obj *DNSConfig) {
+			if len(obj.ClusterDomain) == 0 {
+				obj.ClusterDomain = "cluster.local"
```

Comment:
Ugh, we need to discuss that.  Leave for now, we're considering
changing the default to xip.io